test.js

```diff
@@ -737,52 +737,109 @@ test('Handles non-existing command, shell', async t => {
-test('Can pass arguments to local npm binaries, ,', testLocalBinary, ',');
-test('Can pass arguments to local npm binaries, space', testLocalBinary, ' ');
-test('Can pass arguments to local npm binaries, *', testLocalBinary, '*');
-test('Can pass arguments to local npm binaries, ?', testLocalBinary, '?');
+test('options.preferLocal true can pass arguments to local npm binaries, "', testLocalBinary, '"');
+test('options.preferLocal true can pass arguments to local npm binaries, \\', testLocalBinary, '\\');
+test('options.preferLocal true can pass arguments to local npm binaries, \\.', testLocalBinary, '\\.');
+test('options.preferLocal true can pass arguments to local npm binaries, \\"', testLocalBinary, '\\"');
+test('options.preferLocal true can pass arguments to local npm binaries, \\\\"', testLocalBinary, '\\\\"');
+test('options.preferLocal true can pass arguments to local npm binaries, a b', testLocalBinary, 'a b');
+test('options.preferLocal true can pass arguments to local npm binaries, \'.\'', testLocalBinary, '\'.\'');
+test('options.preferLocal true can pass arguments to local npm binaries, "."', testLocalBinary, '"."');
+test('options.preferLocal true can pass arguments to local npm binaries, (', testLocalBinary, '(');
+test('options.preferLocal true can pass arguments to local npm binaries, )', testLocalBinary, ')');
+test('options.preferLocal true can pass arguments to local npm binaries, ]', testLocalBinary, ']');
+test('options.preferLocal true can pass arguments to local npm binaries, [', testLocalBinary, '[');
+test('options.preferLocal true can pass arguments to local npm binaries, %', testLocalBinary, '%');
+test('options.preferLocal true can pass arguments to local npm binaries, %1', testLocalBinary, '%1');
+test('options.preferLocal true can pass arguments to local npm binaries, !', testLocalBinary, '!');
+test('options.preferLocal true can pass arguments to local npm binaries, ^', testLocalBinary, '^');
+test('options.preferLocal true can pass arguments to local npm binaries, `', testLocalBinary, '`');
+test('options.preferLocal true can pass arguments to local npm binaries, <', testLocalBinary, '<');
+test('options.preferLocal true can pass arguments to local npm binaries, >', testLocalBinary, '>');
+test('options.preferLocal true can pass arguments to local npm binaries, &', testLocalBinary, '&');
+test('options.preferLocal true can pass arguments to local npm binaries, |', testLocalBinary, '|');
+test('options.preferLocal true can pass arguments to local npm binaries, ;', testLocalBinary, ';');
+test('options.preferLocal true can pass arguments to local npm binaries, ,', testLocalBinary, ',');
+test('options.preferLocal true can pass arguments to local npm binaries, space', testLocalBinary, ' ');
+test('options.preferLocal true can pass arguments to local npm binaries, *', testLocalBinary, '*');
+test('options.preferLocal true can pass arguments to local npm binaries, ?', testLocalBinary, '?');
```

Comment:
Doesn't have to be this PR, but maybe pass in the repeatable part of the titles here as a variable to reduce duplication.